deeplake/core/tensor.py

```diff
@@ -318,6 +318,22 @@ def extend(
         dataset_written(self.dataset)
         self.invalidate_libdeeplake_dataset()
 
+    @invalid_view_op
+    def _extend_with_paths(
+        self,
+        samples: Union[np.ndarray, Sequence[InputSample], "Tensor"],
+        progressbar: bool = False,
+    ):
```

Comment:
add a link check at the top of this method, similar to `def _linked_sample(self):`